packages/ketcher-core/src/application/editor/operations/rgroupAttachmentPoint/RGroupAttachmentPointAdd.ts

```diff
@@ -0,0 +1,60 @@
+import { ReStruct, ReRGroupAttachmentPoint } from 'application/render';
+import assert from 'assert';
+import {
+  RGroupAttachmentPoint,
+  RGroupAttachmentPointType,
+} from 'domain/entities';
+import { RGroupAttachmentPointRemove } from '.';
+import { OperationPriority, OperationType } from '../OperationType';
+import BaseOperation from '../base';
+
+type Data = {
+  atomId: number;
+  attachmentPointType: RGroupAttachmentPointType;
+  attachmentPointId?: number;
+};
+
+class RGroupAttachmentPointAdd extends BaseOperation {
+  data: Data;
+
+  constructor(data: Data) {
+    super(
+      OperationType.R_GROUP_ATTACHMENT_POINT_ADD,
+      OperationPriority.R_GROUP_ATTACHMENT_POINT_ADD,
+    );
+    this.data = data;
+  }
+
+  execute(restruct: ReStruct) {
+    const { atomId, attachmentPointType } = this.data;
+    const item = new RGroupAttachmentPoint(atomId, attachmentPointType);
```

Comment:
I would suggest to rename 'item' to 'newAttachmentPoint' or something like that. Item is confusing a bit

Reply:
Thank you. Fixed